scripts/tasks/4_ICA.ipynb

```diff
@@ -0,0 +1,377 @@
+{
```

Comment:
<p>I'm going to guess those filenames raise a warning on the naming convention since they don't end in <code>_ica.fif</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mscheltienne/eeg-flow/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>ok fixed thanks</p>